apps/web/src/pages/integrations/components/NovuIntegrationCard.tsx

```diff
@@ -0,0 +1,145 @@
+import { ChannelTypeEnum, IConfigCredentials } from '@novu/shared';
+import styled from '@emotion/styled';
+import { Group, useMantineColorScheme } from '@mantine/core';
+import { colors, shadows } from '../../../design-system';
+import { CardStatusBar } from './CardStatusBar';
+import { IIntegratedProvider } from '../IntegrationsStorePage';
+import { LimitBar } from './LimitBar';
+
+export function NovuIntegrationCard({
+  provider,
+  onConnectClick = undefined,
+  ribbonText = 'Starting provider',
```

Comment:
```suggestion
  ribbonText = 'Test Provider',
```

Reply:
I agree with @oba2311 comment "it is unclear whether we are testing it or them" but I also do not like "Free" so my suggestion is to have something else... @p-fernandez any input here?

Reply:
Is trail provider better?

Reply:
I am not going to make anyone happy: `Limited free trial provider`. 😂